.github/workflows/trigger-automation-repo.yml

```diff
@@ -0,0 +1,18 @@
+name: Notify Automation Workflow on PR creation
+
+on:
+  pull_request:
+    branches:
+      - main
+
+jobs:
+  notify-automation:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Trigger Automation repo
+        run: |
```

Comment:
Instead of doing it like this, it would be more durable to use https://github.com/peter-evans/repository-dispatch